### PR TITLE
Align NMS code with upstream

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
run tests

**Review by Ilya-Krylov — APPROVED** [review]

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

File: mmdet/utils/deployment/__init__.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions
 # and limitations under the License.
 
-from .onnxruntime_backend import ModelONNXRuntime
-from .openvino_backend import ModelOpenVINO, DetectorOpenVINO
-from .symbolic import register_extra_symbolics
-from .tracer_stubs import TracerStub
+# from .onnxruntime_backend import ModelONNXRuntime
```
Comment (Ilya-Krylov) [inline mmdet/utils/deployment/__init__.py]: ?

File: mmdet/models/dense_heads/rpn_head.py [hunk 2]
```diff
@@ -2,16 +2,17 @@
...
-from mmdet.core.post_processing import multiclass_nms
-from mmdet.core.bbox.coder.delta_xywh_bbox_coder import delta2bbox
-from mmdet.core.utils.misc import topk
+from ...core.utils.misc import topk
 from ..builder import HEADS
 from .anchor_head import AnchorHead
 from .rpn_test_mixin import RPNTestMixin
 
 
+
```
Comment (Ilya-Krylov) [inline mmdet/models/dense_heads/rpn_head.py]: looks like redundant lines

File: mmdet/utils/deployment/__init__.py (hunk 1 above)
Comment (druzhkov-paul) [reply mmdet/utils/deployment/__init__.py]: Circular dependencies were present somewhere in here.

File: mmdet/models/dense_heads/rpn_head.py (hunk 2 above)
Comment (druzhkov-paul) [reply mmdet/models/dense_heads/rpn_head.py]: Yes, it has already been removed. Please review the latest version.

**Pull request merged**